Frontend/sky_line/app/navbar/navbar.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { InputGroup, Button, Container, Form, Navbar, Nav, NavDropdown, Modal, ListGroup } from 'react-bootstrap';
+import { Link } from 'react-router-dom'; // Import Link from react-router-dom
+import '../navbar/navbar.css';
+
+const NavbarComponent: React.FC = () => {
+  const [showNotification, setShowNotification] = useState(false);
+
+  const handleNotificationClick = () => {
+    setShowNotification(!showNotification);
+  };
+
+  return (
+    <>
+      <Navbar expand="lg" className="bg-body-tertiary">
+        <Container fluid>
+          <Navbar.Brand href="#">SKYLINE</Navbar.Brand>
+          <Navbar.Toggle aria-controls="navbarScroll" />
+          <Navbar.Collapse id="navbarScroll" className="justify-content-center">
+            <Nav className="me-auto my-2 my-lg-0" navbarScroll>
+              <Nav.Link href="/login">Home</Nav.Link>
+              <Nav.Link href="/support">Support</Nav.Link>
```

Comment:
nav.link is not good because it makes the web page reload and should use Link or router.push("/login") it works efficiently.